app/templating/view_context.py

```diff
@@ -152,40 +159,46 @@ def _build_calculated_summary_section_list(schema, rendered_block, current_locat
     return [section]
 
 
-def _remove_unwanted_questions_answers(block, answers_in_block, answer_ids_to_keep):
+def _remove_unwanted_questions_answers(block, answers_in_block, answer_ids_to_keep, answer_store, metadata, schema):
+    """
+    Evaluates questions in a block and removes any questions not containing a relevant answer
+    """
+    block_question = choose_question_to_display(block, schema, answer_store, metadata)
+
     reduced_block = block.copy()
     questions_to_keep = [
         answer['parent_id'] for answer in answers_in_block.values() if answer['id'] in answer_ids_to_keep
```

Comment:
Can this be more than 1?

Reply:
What do you mean?

Reply:
Was referring to the fact that there would only be 1 question. But seems fine to be a list of one.